examples/blackjack/play.py

```diff
@@ -0,0 +1,21 @@
+import game
+import gym
+
+gym.logger.set_level(40)
+
+
+SEED = 42
+
+if __name__ == "__main__":
+
+    env = gym.make('BlackjackEnv')
+    state, _ = env.reset(seed=SEED)
+    env.render()
+
+    done = False
+    while not done:
+
+        act = int(input("Type 1 for hit or 0 for stick: "))
+        print(f"action = {act}")
+        state, reward, done, _ = env.step(act)
```

Comment:
Używasz starego czy nowego API gym? Jeśli nowego, to tutaj brakuje chyba jednego pola, a jak starego to polecam przejść na nowe (ponieważ jest nowe xd). Ewentualnie prosty symulator można przekształcić na stare, ale ważne żeby było spójnie.

Reply:
Okey, właśnie czytałem docu `step()` i nie wiedziałem za bardzo o co chodzi z tym * truncated* szczególnie że pamiętam jak ostatnio na PUMy gdy pisałem swoje środowisko to tego jeszcze nie było. Ale dzięki za tą uwagę, dostosuję się do nowego API.